### ISSUE TITLE
Where is best to track designed sound prototypes and originals?

### ISSUE THREAD

**Issue by zepumph** [phetsims/tambo#127]
Tracking discussion from https://github.com/phetsims/ratio-and-proportion/issues/191#issuecomment-717328957 in a new issues. Basically I was making the case for having as few files outside of the "PhET cloud" as possible, and to try to create a place where it is easy to make sure that filenames for sounds checked into `git` match the originals that @Ashton-Morris has to work on. This will increase maintainability if we ever need to go back and make tweaks. 

To me the central ground truth should be on github, which I think we do well by having `.wav` files in the `assets/` folder. @Ashton-Morris mentioned that this work may be done as part of the sound board he is working with @jbphet on. I'll pass it over to the sound designers to take it from here.

**Comment by Ashton-Morris** [comment]
From what I remember from our meeting a few weeks ago @jbphet was suggesting that it was too messy to keep iteration audio files in some repo somewhere. Can you refresh my mind if we decided that this wasn't a priority?

**Comment by jbphet** [comment]
So far, the process has been that @Ashton-Morris provides us with `.mp3` during the iteration phase and, once we've decided upon a sound, we put a `.wav` version of that sound in the `assets` directory, encode that file based on the recommendations in https://github.com/phetsims/tambo/blob/master/doc/encoding-audio.md, and make sure that the encoded file name matches the `.wav` file except, of course, for the file type.

In https://github.com/phetsims/tambo/issues/52 we decided that we can't keep the raw files from the sound tools because of various licensing issues, and they probably wouldn't do us much good anyway, and we opted to keep just the `.wav` file as the asset.

While it would be nice to keep all of the assets around in `.wav` format during the entire sound implementation process, I for one don't believe that it is worth the time and effort to do so.  We tend to iterate on the sounds a lot, and requiring developers to keep an asset for every one of these only to delete 90% of them later doesn't feel to me like it's the right cost/benefit tradeoff.

Back to @zepumph for further consideration.
